testing/av_test.c

```diff
@@ -22,6 +22,20 @@
  *   -lopencv_highgui -lopencv_imgproc -lsndfile -pthread -lvpx -lopus -lsodium -lportaudio
  */
 
+// XXX: Hack because toxav doesn't really expose ring_buffer, but this av test
+// uses it.
+#define RingBuffer TestRingBuffer
+#define rb_full  test_rb_full
```

Comment:
They aren't used, but if linking statically, not renaming them will cause multiple definition errors. We need to rename all of them.